### ISSUE TITLE
Another weird Couchpotato API call bug?

### ISSUE THREAD

**Issue by slyde56** [clinton-hall/nzbToMedia#661]
Hi Clinton,

I've got another weird bug coming from Couchpotato. I think the only thing it's affecting is that the downloads aren't reported as being finished (successful or failed). The renamer is being run, though, and the movies go where they're supposed to go.

It's just a normal Windows PC setup all on the same box. I've checked the API key and host in CouchPotato and autoProcessMedia.cfg, but I still get a "no json object could be decoded" error. Thanks for your help!

```
[22:06:10] [INFO]::MAIN: Checking database structure...
[22:06:10] [DEBUG]::MAIN: Checking Initial Schema database upgrade
[22:06:10] [DEBUG]::MAIN: InitialSchema upgrade not required
[22:06:10] [DEBUG]::MAIN: Checking if we can use git commands: git version
[22:06:10] [DEBUG]::MAIN: Executing git version with your shell in Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia
[22:06:10] [DEBUG]::MAIN: git version : returned successful
[22:06:10] [DEBUG]::MAIN: Using: git
[22:06:10] [DEBUG]::MAIN: Executing git symbolic-ref -q HEAD with your shell in Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia
[22:06:10] [DEBUG]::MAIN: git symbolic-ref -q HEAD : returned successful
[22:06:10] [INFO]::MAIN: Checking if git needs an update
[22:06:10] [DEBUG]::MAIN: Executing git rev-parse HEAD with your shell in Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia
[22:06:10] [DEBUG]::MAIN: git rev-parse HEAD : returned successful
[22:06:10] [DEBUG]::MAIN: Executing git fetch origin with your shell in Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia
[22:06:11] [DEBUG]::MAIN: git fetch origin : returned successful
[22:06:11] [DEBUG]::MAIN: Executing git rev-parse --verify --quiet "@{upstream}" with your shell in Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia
[22:06:11] [DEBUG]::MAIN: git rev-parse --verify --quiet "@{upstream}" : returned successful
[22:06:11] [DEBUG]::MAIN: Executing git rev-list --left-right "@{upstream}"...HEAD with your shell in Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia
[22:06:11] [DEBUG]::MAIN: git rev-list --left-right "@{upstream}"...HEAD : returned successful
[22:06:11] [DEBUG]::MAIN: cur_commit = 90734d3fab19256b46b0b73eee5a348feb89d3f6 % (newest_commit)= 90734d3fab19256b46b0b73eee5a348feb89d3f6, num_commits_behind = 0, num_commits_ahead = 0
[22:06:11] [INFO]::MAIN: No update needed
[22:06:11] [INFO]::MAIN: nzbToMedia Version:90734d3fab19256b46b0b73eee5a348feb89d3f6 Branch:nightly (Windows 7)
[22:06:11] [INFO]::MAIN: #########################################################
[22:06:11] [INFO]::MAIN: ## ..::[nzbToMedia.pyc]::.. ##
[22:06:11] [INFO]::MAIN: #########################################################
[22:06:11] [DEBUG]::MAIN: Options passed into nzbToMedia: ['Q:\\Program Files (x86)\\SABnzbd\\scripts\\nzbToMedia\\nzbToCouchPotato.py', 'Q:\\Downloads\\Movies\\Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227)', 'Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227).nzb', 'Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227)', '', 'couchpotato', 'alt.binaries.mom', '0']
[22:06:11] [INFO]::MAIN: Script triggered from SABnzbd
[22:06:11] [DEBUG]::MAIN: Searching for nzoid from SAbnzbd ...
[22:06:11] [DEBUG]::MAIN: Adding NZB download info for directory Q:\Downloads\Movies\Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227) to database
[22:06:11] [INFO]::MAIN: Auto-detected SECTION:CouchPotato
[22:06:11] [DEBUG]::MAIN: Checking for archives to extract in directory: Q:\Downloads\Movies\Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227)
[22:06:11] [DEBUG]::MAIN: EXTRACTOR: Unknown file type: .1
[22:06:11] [INFO]::MAIN: Calling CouchPotato:couchpotato to post-process:Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227).nzb
[22:06:12] [INFO]::MAIN: Attemping imdbID lookup for Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227).nzb
[22:06:12] [INFO]::MAIN: Searching folder and file names for imdbID ...
[22:06:12] [INFO]::MAIN: Found imdbID [tt2262227]
[22:06:12] [DEBUG]::MAIN: Opening URL: http://localhost:5050/api/574eb150cf9f4e1da29bcdefdcacbe3b/media.get with PARAMS: {'id': 'tt2262227'}
[22:06:13] [ERROR]::MAIN: CouchPotato returned the following non-json data
[22:06:13] [ERROR]::MAIN: 
[22:06:13] [ERROR]::MAIN: <!doctype html>
[22:06:13] [ERROR]::MAIN: <html>
[22:06:13] [ERROR]::MAIN: <head>
[22:06:13] [ERROR]::MAIN: <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"/>
[22:06:13] [ERROR]::MAIN: <meta name="apple-mobile-web-app-capable" content="yes">
[22:06:13] [ERROR]::MAIN: <meta name="mobile-web-app-capable" content="yes">
[22:06:13] [ERROR]::MAIN: <meta http-equiv="X-UA-Compatible" content="IE=edge" />
[22:06:13] [ERROR]::MAIN: 
[22:06:13] [ERROR]::MAIN: <link rel="stylesheet" href="/couchpotato/minified/front.css?1419055906" type="text/css">
[22:06:13] [ERROR]::MAIN: 
[22:06:13] [ERROR]::MAIN: <script type="text/javascript" src="/couchpotato/minified/front.js?1419055917"></script>
[22:06:13] [ERROR]::MAIN: 
[22:06:13] [ERROR]::MAIN: <script type="text/javascript" src="/couchpotato/minified/head.js?1419055918"></script>
[22:06:13] [ERROR]::MAIN: 
[22:06:13] [ERROR]::MAIN: <link rel="stylesheet" href="/couchpotato/minified/head.css?1419055909" type="text/css">
[22:06:13] [ERROR]::MAIN: <link href="/couchpotato/static/images/favicon.ico" rel="icon" type="image/x-icon" />
[22:06:13] [ERROR]::MAIN: <link rel="apple-touch-icon" href="/couchpotato/static/images/homescreen.png" />
[22:06:13] [ERROR]::MAIN: <script type="text/javascript" src="https://www.youtube.com/player_api" defer="defer"></script>
[22:06:13] [ERROR]::MAIN: <script type="text/javascript">
[22:06:13] [ERROR]::MAIN: window.addEvent('load', function(){
[22:06:13] [ERROR]::MAIN: if(window.getSize().x <= 480)
[22:06:13] [ERROR]::MAIN: setTimeout(function(){
[22:06:13] [ERROR]::MAIN: window.scrollTo(0, 1);
[22:06:13] [ERROR]::MAIN: window.scrollTo(0, 0);
[22:06:13] [ERROR]::MAIN: }, 100);
[22:06:13] [ERROR]::MAIN: });
[22:06:13] [ERROR]::MAIN: window.addEvent('domready', function() {
[22:06:13] [ERROR]::MAIN: new Uniform();
[22:06:13] [ERROR]::MAIN: Api.setup({
[22:06:13] [ERROR]::MAIN: 'url': "/couchpotato/api/574eb150cf9f4e1da29bcdefdcacbe3b/",
[22:06:13] [ERROR]::MAIN: 'path_sep': "\\",
[22:06:13] [ERROR]::MAIN: 'is_remote': false
[22:06:13] [ERROR]::MAIN: });
[22:06:13] [ERROR]::MAIN: $(document.body).set('data-api', window.location.protocol + '//' + window.location.host + Api.createUrl().replace('/default/', '/'));
[22:06:13] [ERROR]::MAIN: // Catch errors
[22:06:13] [ERROR]::MAIN: window.onerror = function(message, file, line){
[22:06:13] [ERROR]::MAIN: p(message, file, line);
[22:06:13] [ERROR]::MAIN: Api.request('logging.log', {
[22:06:13] [ERROR]::MAIN: 'data': {
[22:06:13] [ERROR]::MAIN: 'type': 'error',
[22:06:13] [ERROR]::MAIN: 'message': Browser.name + ' ' + Browser.version + ': \n' + message,
[22:06:13] [ERROR]::MAIN: 'page': window.location.href.replace(window.location.host, 'HOST'),
[22:06:13] [ERROR]::MAIN: 'file': file.replace(window.location.host, 'HOST'),
[22:06:13] [ERROR]::MAIN: 'line': line
[22:06:13] [ERROR]::MAIN: }
[22:06:13] [ERROR]::MAIN: });
[22:06:13] [ERROR]::MAIN: return true;
[22:06:13] [ERROR]::MAIN: }
[22:06:13] [ERROR]::MAIN: Quality.setup({
[22:06:13] [ERROR]::MAIN: 'profiles': [{"core": 0, "_rev": "00017af1", "finish": [1, 1, 1, 1], "_id": "4c6495f171d04862ba4409c725c45521", "_t": "profile", "label": "Best", "qualities": ["720p", "1080p", "brrip", "dvdrip"], "wait_for": [0, 0, 0, 0], "order": -2}, {"core": 0, "_rev": "00018cf5", "finish": [1, 1], "_id": "337e9b202e184e4a980d3d7bbb8a8201", "_t": "profile", "label": "HD", "qualities": ["720p", "1080p"], "wait_for": [0, 0], "order": -1}, {"core": true, "_rev": "0002ab56", "finish": [true], "hide": false, "_id": "39a10e4618374007a0df40aa069de45b", "_t": "profile", "label": "BR-Disk", "qualities": ["bd50"], "wait_for": [0], "order": 0}, {"core": 0, "_rev": "000155ee", "finish": [1, 1], "_id": "674eb8b31b2744b1857abd4ba9947b21", "_t": "profile", "label": "SD", "qualities": ["dvdrip", "dvdr"], "wait_for": [0, 0], "order": 0}, {"core": true, "_rev": "00025fb8", "finish": [true], "hide": false, "_id": "d45922e667fd45578d4218c8b0c13948", "_t": "profile", "label": "1080p", "qualities": ["1080p"], "wait_for": [0], "order": 1}, {"core": true, "_rev": "00020151", "finish": [true], "hide": false, "_id": "e395b075e4da43d6895b1be7485c6329", "_t": "profile", "label": "720p", "qualities": ["720p"], "wait_for": [0], "order": 2}, {"core": true, "_rev": "00025b16", "finish": [true], "hide": false, "_id": "825d3310b3564bce8bc259836d269737", "_t": "profile", "label": "BR-Rip", "qualities": ["brrip"], "wait_for": [0], "order": 3}, {"core": true, "_rev": "00026b59", "finish": [true], "hide": false, "_id": "b7a0dfb64fa84e83bb7163833b1c990a", "_t": "profile", "label": "DVD-R", "qualities": ["dvdr"], "wait_for": [0], "order": 4}, {"core": true, "_rev": "0002c49f", "finish": [true], "hide": false, "_id": "b32af5b0a3e447c8a174658b7deceaea", "_t": "profile", "label": "DVD-Rip", "qualities": ["dvdrip"], "wait_for": [0], "order": 5}, {"core": true, "_rev": "000279f5", "finish": [true], "hide": false, "_id": "186237100816406faab1fa3a6c853507", "_t": "profile", "label": "Screener", "qualities": ["scr"], "wait_for": [0], "order": 6}, {"core": true, "_rev": "00027d57", "finish": [true], "hide": false, "_id": "04d55372785d475fb3d608d4c1c92c23", "_t": "profile", "label": "R5", "qualities": ["r5"], "wait_for": [0], "order": 7}, {"core": true, "_rev": "0002f640", "finish": [true], "hide": false, "_id": "233bbf78f79c4c1c982f3f536bffd178", "_t": "profile", "label": "TeleCine", "qualities": ["tc"], "wait_for": [0], "order": 8}, {"core": true, "_rev": "00028460", "finish": [true], "hide": false, "_id": "9f59f8f7dc604ad99fd60a5e03663d90", "_t": "profile", "label": "TeleSync", "qualities": ["ts"], "wait_for": [0], "order": 9}, {"core": true, "_rev": "0002b42b", "finish": [true], "hide": false, "_id": "9d6b102636004f91b394da6d24c9cdf7", "_t": "profile", "label": "Cam", "qualities": ["cam"], "wait_for": [0], "order": 10}],
[22:06:13] [ERROR]::MAIN: 'qualities': [{"_id": "e233ec7cc53b45ef8c37011ff5956b5b", "tags": ["bdmv", "certificate", ["complete", "bluray"], "avc", "mvc"], "_rev": "0002aad8", "_t": "quality", "size_max": 60000, "size_min": 15000, "label": "BR-Disk", "order": 0, "ext": ["iso", "img"], "is_3d": false, "allow_3d": true, "allow": ["1080p"], "alternative": ["bd25", ["br", "disk"]], "identifier": "bd50", "median_size": 40000, "hd": true, "size": [20000, 60000]}, {"_rev": "0002e0dd", "identifier": "1080p", "size_min": 4000, "tags": ["m2ts", "x264", "h264"], "_id": "8e78451ba9b5470091a5425c03c41975", "_t": "quality", "size_max": 20000, "label": "1080p", "height": 1080, "order": 1, "width": 1920, "ext": ["mkv", "m2ts", "ts"], "is_3d": false, "allow": [], "alternative": [], "allow_3d": true, "median_size": 10000, "hd": true, "size": [4000, 20000]}, {"_rev": "000262b2", "identifier": "720p", "size_min": 3000, "tags": ["x264", "h264"], "_id": "31a06fd1730c4eb38be43bf5980ac13d", "_t": "quality", "size_max": 10000, "label": "720p", "height": 720, "order": 2, "width": 1280, "ext": ["mkv", "ts"], "is_3d": false, "allow": [], "alternative": [], "allow_3d": true, "median_size": 5500, "hd": true, "size": [3000, 10000]}, {"_id": "8d6eb768aa024bb9bb0b04d39d5ed5a9", "tags": ["webdl", ["web", "dl"]], "_rev": "0002f89a", "_t": "quality", "size_max": 7000, "size_min": 700, "label": "BR-Rip", "order": 3, "ext": ["mp4", "avi"], "is_3d": false, "allow_3d": true, "allow": ["720p", "1080p"], "alternative": ["bdrip", ["br", "rip"], "hdtv", "hdrip"], "identifier": "brrip", "median_size": 2000, "hd": true, "size": [700, 7000]}, {"_id": "243c07fd8d2540fb87f04db0caa0f0dd", "tags": ["pal", "ntsc", "video_ts", "audio_ts", ["dvd", "r"], "dvd9"], "_rev": "0002153c", "_t": "quality", "size_max": 10000, "size_min": 3000, "label": "DVD-R", "order": 4, "ext": ["iso", "img", "vob"], "is_3d": false, "allow": [], "alternative": ["br2dvd", ["dvd", "r"]], "identifier": "dvdr", "median_size": 4500, "size": [3000, 10000]}, {"_id": "98f0a9abbc6a46aaa423cbb9289f699f", "tags": [["dvd", "rip"], ["dvd", "xvid"], ["dvd", "divx"]], "_rev": "0002a499", "_t": "quality", "size_max": 2400, "size_min": 600, "label": "DVD-Rip", "order": 5, "width": 720, "ext": ["avi"], "is_3d": false, "allow": [], "alternative": [["dvd", "rip"]], "identifier": "dvdrip", "median_size": 1500, "size": [600, 2400]}, {"_id": "4ff7c177eb7b4112bf406ea4b5535fd3", "tags": [], "_rev": "0002f085", "_t": "quality", "size_max": 1600, "size_min": 600, "label": "Screener", "order": 6, "ext": [], "is_3d": false, "allow": ["dvdr", "dvdrip", "720p", "1080p"], "alternative": ["screener", "dvdscr", "ppvrip", "dvdscreener", "hdscr", "webrip", ["web", "rip"]], "identifier": "scr", "median_size": 700, "size": [600, 1600]}, {"_rev": "000217c6", "order": 7, "size_min": 600, "is_3d": false, "alternative": ["r6"], "size": [600, 1000], "identifier": "r5", "_t": "quality", "size_max": 1000, "label": "R5", "ext": [], "allow": ["dvdr", "720p", "1080p"], "_id": "35aeaed706ce4ff4b7a9dce3b7dba133", "median_size": 700}, {"_rev": "000274dc", "order": 8, "size_min": 600, "alternative": ["telecine"], "size": [600, 1000], "identifier": "tc", "_t": "quality", "size_max": 1000, "label": "TeleCine", "ext": [], "allow": ["720p", "1080p"], "_id": "775bb74812ba4663bc22ebe8d6912e03", "median_size": 700}, {"_rev": "0002e0c2", "order": 9, "size_min": 600, "is_3d": false, "alternative": ["telesync", "hdts"], "size": [600, 1000], "identifier": "ts", "_t": "quality", "size_max": 1000, "label": "TeleSync", "ext": [], "allow": ["720p", "1080p"], "_id": "bf4a862516144cc4bad3ff60d7689873", "median_size": 700}, {"_rev": "0002b035", "order": 10, "size_min": 600, "is_3d": false, "alternative": ["camrip", "hdcam"], "size": [600, 1000], "identifier": "cam", "_t": "quality", "size_max": 1000, "label": "Cam", "ext": [], "allow": ["720p", "1080p"], "_id": "ed8c343c6a3644a2a0d90f290039d0fc", "median_size": 700}]
[22:06:13] [ERROR]::MAIN: });
[22:06:13] [ERROR]::MAIN: CategoryList.setup([]);
[22:06:13] [ERROR]::MAIN: App.setup({
[22:06:13] [ERROR]::MAIN: 'base_url': "/couchpotato/",
[22:06:13] [ERROR]::MAIN: 'args': "['--quiet']",
[22:06:13] [ERROR]::MAIN: 'options': "Namespace(config_file='X:\\\\Users\\\\Sean J\\\\AppData\\\\Roaming\\\\CouchPotato\\\\settings.conf', console_log=False, daemon=False, data_dir=None, debug=False, pid_file='X:\\\\Users\\\\Sean J\\\\AppData\\\\Roaming\\\\CouchPotato\\\\couchpotato.pid', quiet=True)",
[22:06:13] [ERROR]::MAIN: 'app_dir': "X:\\Users\\Sean J\\AppData\\Roaming\\CouchPotato\\application\\appdata\\CouchPotato-2.6.1.win32",
[22:06:13] [ERROR]::MAIN: 'data_dir': "X:\\Users\\Sean J\\AppData\\Roaming\\CouchPotato",
[22:06:13] [ERROR]::MAIN: 'pid': "4280 ",
[22:06:13] [ERROR]::MAIN: 'userscript_version': 25
[22:06:13] [ERROR]::MAIN: });
[22:06:13] [ERROR]::MAIN: })
[22:06:13] [ERROR]::MAIN: 
[22:06:13] [ERROR]::MAIN: </script>
[22:06:13] [ERROR]::MAIN: <title>CouchPotato</title>
[22:06:13] [ERROR]::MAIN: </head>
[22:06:13] [ERROR]::MAIN: <body></body>
[22:06:13] [ERROR]::MAIN: </html>
[22:06:13] [DEBUG]::MAIN: calling command: Q:\Program Files (x86)\SABnzbd\scripts\ffmpeg\bin\ffprobe.exe -v quiet -print_format json -show_format -show_streams -show_error Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia\tests\test.mp4
[22:06:13] [INFO]::TRANSCODER: Checking [Movie.2014.REPACK.720p.BluRay.x264-GECKOS.mkv] for corruption, please stand by ...
[22:06:13] [DEBUG]::MAIN: calling command: Q:\Program Files (x86)\SABnzbd\scripts\ffmpeg\bin\ffprobe.exe -v quiet -print_format json -show_format -show_streams -show_error Q:\Downloads\Movies\Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227)\Movie.2014.REPACK.720p.BluRay.x264-GECKOS.mkv
[22:06:13] [INFO]::TRANSCODER: SUCCESS: [Movie.2014.REPACK.720p.BluRay.x264-GECKOS.mkv] has no corruption.
[22:06:14] [DEBUG]::SUBTITLES: Attempting to download subtitles for Q:\Downloads\Movies\Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227)\Movie.2014.REPACK.720p.BluRay.x264-GECKOS.mkv
[22:06:18] [DEBUG]::COUCHPOTATO: Opening URL: http://localhost:5050/api/574eb150cf9f4e1da29bcdefdcacbe3b/renamer.scan with PARAMS: {'media_folder': 'Q:\\Downloads\\Movies\\Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227)'}
[22:06:18] [POSTPROCESS]::COUCHPOTATO: Starting renamer scan for Movie.2014.Repack.720p.BluRay.x264-GECKOS.cp(tt2262227).nzb
Traceback (most recent call last):
  File "Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia\nzbToCouchPotato.py", line 247, in <module>
    result = nzbToMedia.main(sys.argv, section)
  File "Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia\nzbToMedia.py", line 698, in main
    download_id='')
  File "Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia\nzbToMedia.py", line 583, in process
    inputCategory, failureLink)
  File "Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia\nzbtomedia\autoProcess\autoProcessMovie.py", line 242, in process
    result = r.json()
  File "Q:\Program Files (x86)\SABnzbd\scripts\nzbToMedia\libs\requests\models.py", line 759, in json
    return json.loads(self.text, **kwargs)
  File "C:\Python27\lib\json\__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "C:\Python27\lib\json\decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Python27\lib\json\decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded

**Comment by clinton-hall** [comment]
what happens if you post
http://localhost:5050/api/574eb150cf9f4e1da29bcdefdcacbe3b/media.get/?id=tt2262227
into your local browser?

**Comment by slyde56** [comment]
This html output:
```html
<!DOCTYPE html>
<!-- saved from url=(0082)http://localhost:5050/couchpotato/cf9f4e1da29bcdefdcacbe3b/media.get/?id=tt2262227 -->
<html slick-uniqueid="25"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
<meta name="apple-mobile-web-app-capable" content="yes">
<meta name="mobile-web-app-capable" content="yes">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<link rel="stylesheet" href="./CouchPotato_files/front.css" type="text/css">

<script id="www-widgetapi-script" src="./CouchPotato_files/www-widgetapi.js" async=""></script><script type="text/javascript" src="./CouchPotato_files/front.js"></script><style></style>

<script type="text/javascript" src="./CouchPotato_files/head.js"></script>

<link rel="stylesheet" href="./CouchPotato_files/head.css" type="text/css">
<link href="http://localhost:5050/couchpotato/static/images/favicon.ico" rel="icon" type="image/x-icon">
<link rel="apple-touch-icon" href="http://localhost:5050/couchpotato/static/images/homescreen.png">
<script type="text/javascript" src="./CouchPotato_files/player_api" defer="defer"></script>
<script type="text/javascript">
window.addEvent('load', function(){
if(window.getSize().x <= 480)
setTimeout(function(){
window.scrollTo(0, 1);
window.scrollTo(0, 0);
}, 100);
});
window.addEvent('domready', function() {
new Uniform();
Api.setup({
'url': "/couchpotato/api/574eb150cf9f4e1da29bcdefdcacbe3b/",
'path_sep': "\\",
'is_remote': false
});
$(document.body).set('data-api', window.location.protocol + '//' + window.location.host + Api.createUrl().replace('/default/', '/'));
// Catch errors
window.onerror = function(message, file, line){
p(message, file, line);
Api.request('logging.log', {
'data': {
'type': 'error',
'message': Browser.name + ' ' + Browser.version + ': \n' + message,
'page': window.location.href.replace(window.location.host, 'HOST'),
'file': file.replace(window.location.host, 'HOST'),
'line': line
}
});
return true;
}
Quality.setup({
'profiles': [{"core": 0, "_rev": "00017af1", "finish": [1, 1, 1, 1], "_id": "4c6495f171d04862ba4409c725c45521", "_t": "profile", "label": "Best", "qualities": ["720p", "1080p", "brrip", "dvdrip"], "wait_for": [0, 0, 0, 0], "order": -2}, {"core": 0, "_rev": "00018cf5", "finish": [1, 1], "_id": "337e9b202e184e4a980d3d7bbb8a8201", "_t": "profile", "label": "HD", "qualities": ["720p", "1080p"], "wait_for": [0, 0], "order": -1}, {"core": true, "_rev": "0002ab56", "finish": [true], "hide": false, "_id": "39a10e4618374007a0df40aa069de45b", "_t": "profile", "label": "BR-Disk", "qualities": ["bd50"], "wait_for": [0], "order": 0}, {"core": 0, "_rev": "000155ee", "finish": [1, 1], "_id": "674eb8b31b2744b1857abd4ba9947b21", "_t": "profile", "label": "SD", "qualities": ["dvdrip", "dvdr"], "wait_for": [0, 0], "order": 0}, {"core": true, "_rev": "00025fb8", "finish": [true], "hide": false, "_id": "d45922e667fd45578d4218c8b0c13948", "_t": "profile", "label": "1080p", "qualities": ["1080p"], "wait_for": [0], "order": 1}, {"core": true, "_rev": "00020151", "finish": [true], "hide": false, "_id": "e395b075e4da43d6895b1be7485c6329", "_t": "profile", "label": "720p", "qualities": ["720p"], "wait_for": [0], "order": 2}, {"core": true, "_rev": "00025b16", "finish": [true], "hide": false, "_id": "825d3310b3564bce8bc259836d269737", "_t": "profile", "label": "BR-Rip", "qualities": ["brrip"], "wait_for": [0], "order": 3}, {"core": true, "_rev": "00026b59", "finish": [true], "hide": false, "_id": "b7a0dfb64fa84e83bb7163833b1c990a", "_t": "profile", "label": "DVD-R", "qualities": ["dvdr"], "wait_for": [0], "order": 4}, {"core": true, "_rev": "0002c49f", "finish": [true], "hide": false, "_id": "b32af5b0a3e447c8a174658b7deceaea", "_t": "profile", "label": "DVD-Rip", "qualities": ["dvdrip"], "wait_for": [0], "order": 5}, {"core": true, "_rev": "000279f5", "finish": [true], "hide": false, "_id": "186237100816406faab1fa3a6c853507", "_t": "profile", "label": "Screener", "qualities": ["scr"], "wait_for": [0], "order": 6}, {"core": true, "_rev": "00027d57", "finish": [true], "hide": false, "_id": "04d55372785d475fb3d608d4c1c92c23", "_t": "profile", "label": "R5", "qualities": ["r5"], "wait_for": [0], "order": 7}, {"core": true, "_rev": "0002f640", "finish": [true], "hide": false, "_id": "233bbf78f79c4c1c982f3f536bffd178", "_t": "profile", "label": "TeleCine", "qualities": ["tc"], "wait_for": [0], "order": 8}, {"core": true, "_rev": "00028460", "finish": [true], "hide": false, "_id": "9f59f8f7dc604ad99fd60a5e03663d90", "_t": "profile", "label": "TeleSync", "qualities": ["ts"], "wait_for": [0], "order": 9}, {"core": true, "_rev": "0002b42b", "finish": [true], "hide": false, "_id": "9d6b102636004f91b394da6d24c9cdf7", "_t": "profile", "label": "Cam", "qualities": ["cam"], "wait_for": [0], "order": 10}],
'qualities': [{"_id": "e233ec7cc53b45ef8c37011ff5956b5b", "tags": ["bdmv", "certificate", ["complete", "bluray"], "avc", "mvc"], "_rev": "0002aad8", "_t": "quality", "size_max": 60000, "size_min": 15000, "label": "BR-Disk", "order": 0, "ext": ["iso", "img"], "allow_3d": true, "allow": ["1080p"], "alternative": ["bd25", ["br", "disk"]], "identifier": "bd50", "median_size": 40000, "hd": true, "size": [20000, 60000]}, {"_rev": "0002e0dd", "identifier": "1080p", "size_min": 4000, "tags": ["m2ts", "x264", "h264"], "_id": "8e78451ba9b5470091a5425c03c41975", "_t": "quality", "size_max": 20000, "label": "1080p", "height": 1080, "order": 1, "width": 1920, "ext": ["mkv", "m2ts", "ts"], "is_3d": false, "allow": [], "alternative": [], "allow_3d": true, "median_size": 10000, "hd": true, "size": [4000, 20000]}, {"_rev": "000262b2", "identifier": "720p", "size_min": 3000, "tags": ["x264", "h264"], "_id": "31a06fd1730c4eb38be43bf5980ac13d", "_t": "quality", "size_max": 10000, "label": "720p", "height": 720, "order": 2, "width": 1280, "ext": ["mkv", "ts"], "is_3d": false, "allow": [], "alternative": [], "allow_3d": true, "median_size": 5500, "hd": true, "size": [3000, 10000]}, {"_id": "8d6eb768aa024bb9bb0b04d39d5ed5a9", "tags": ["webdl", ["web", "dl"]], "_rev": "0002f89a", "_t": "quality", "size_max": 7000, "size_min": 700, "label": "BR-Rip", "order": 3, "ext": ["mp4", "avi"], "is_3d": false, "allow_3d": true, "allow": ["720p", "1080p"], "alternative": ["bdrip", ["br", "rip"], "hdtv", "hdrip"], "identifier": "brrip", "median_size": 2000, "hd": true, "size": [700, 7000]}, {"_id": "243c07fd8d2540fb87f04db0caa0f0dd", "tags": ["pal", "ntsc", "video_ts", "audio_ts", ["dvd", "r"], "dvd9"], "_rev": "0002153c", "_t": "quality", "size_max": 10000, "size_min": 3000, "label": "DVD-R", "order": 4, "ext": ["iso", "img", "vob"], "is_3d": false, "allow": [], "alternative": ["br2dvd", ["dvd", "r"]], "identifier": "dvdr", "median_size": 4500, "size": [3000, 10000]}, {"_id": "98f0a9abbc6a46aaa423cbb9289f699f", "tags": [["dvd", "rip"], ["dvd", "xvid"], ["dvd", "divx"]], "_rev": "0002a499", "_t": "quality", "size_max": 2400, "size_min": 600, "label": "DVD-Rip", "order": 5, "width": 720, "ext": ["avi"], "is_3d": false, "allow": [], "alternative": [["dvd", "rip"]], "identifier": "dvdrip", "median_size": 1500, "size": [600, 2400]}, {"_id": "4ff7c177eb7b4112bf406ea4b5535fd3", "tags": [], "_rev": "0002f085", "_t": "quality", "size_max": 1600, "size_min": 600, "label": "Screener", "order": 6, "ext": [], "is_3d": false, "allow": ["dvdr", "dvdrip", "720p", "1080p"], "alternative": ["screener", "dvdscr", "ppvrip", "dvdscreener", "hdscr", "webrip", ["web", "rip"]], "identifier": "scr", "median_size": 700, "size": [600, 1600]}, {"_rev": "000217c6", "order": 7, "size_min": 600, "is_3d": false, "alternative": ["r6"], "size": [600, 1000], "identifier": "r5", "_t": "quality", "size_max": 1000, "label": "R5", "ext": [], "allow": ["dvdr", "720p", "1080p"], "_id": "35aeaed706ce4ff4b7a9dce3b7dba133", "median_size": 700}, {"_rev": "000274dc", "order": 8, "size_min": 600, "alternative": ["telecine"], "size": [600, 1000], "identifier": "tc", "_t": "quality", "size_max": 1000, "label": "TeleCine", "ext": [], "allow": ["720p", "1080p"], "_id": "775bb74812ba4663bc22ebe8d6912e03", "median_size": 700}, {"_rev": "0002e0c2", "order": 9, "size_min": 600, "alternative": ["telesync", "hdts"], "size": [600, 1000], "identifier": "ts", "_t": "quality", "size_max": 1000, "label": "TeleSync", "ext": [], "allow": ["720p", "1080p"], "_id": "bf4a862516144cc4bad3ff60d7689873", "median_size": 700}, {"_rev": "0002b035", "order": 10, "size_min": 600, "is_3d": false, "alternative": ["camrip", "hdcam"], "size": [600, 1000], "identifier": "cam", "_t": "quality", "size_max": 1000, "label": "Cam", "ext": [], "allow": ["720p", "1080p"], "_id": "ed8c343c6a3644a2a0d90f290039d0fc", "median_size": 700}]
});
CategoryList.setup([]);
App.setup({
'base_url': "/couchpotato/",
'args': "['--quiet']",
'options': "Namespace(config_file='X:\\\\Users\\\\Sean J\\\\AppData\\\\Roaming\\\\CouchPotato\\\\settings.conf', console_log=False, daemon=False, data_dir=None, debug=False, pid_file='X:\\\\Users\\\\Sean J\\\\AppData\\\\Roaming\\\\CouchPotato\\\\couchpotato.pid', quiet=True)",
'app_dir': "X:\\Users\\Sean J\\AppData\\Roaming\\CouchPotato\\application\\appdata\\CouchPotato-2.6.1.win32",
'data_dir': "X:\\Users\\Sean J\\AppData\\Roaming\\CouchPotato",
'pid': "4396 ",
'userscript_version': 25
});
})

</script>
<title>CouchPotato</title>
</head>
<body data-api="http://localhost:5050/couchpotato/api/574eb150cf9f4e1da29bcdefdcacbe3b/">

<div class="block header"><div><div class="navigation"><a class="foldout icon2 menu"><span class="overlay"></span></a><a href="http://localhost:5050/couchpotato/" class="logo">CouchPotato</a><ul><li class="tab_home"><a href="http://localhost:5050/couchpotato/home/" title="Manage new stuff for things and such">Home</a></li><li class="tab_wanted"><a href="http://localhost:5050/couchpotato/wanted/" title="Gimmy gimmy gimmy!">Wanted</a></li><li class="tab_manage"><a href="http://localhost:5050/couchpotato/manage/" title="Do stuff to your existing movies!">Manage</a></li></ul><a class="backtotop">back to top</a></div><div class="search_form"><div class="input"><input placeholder="Search &amp; add a new media"><a class="icon2"></a></div><div class="results_container"><div class="results"></div><div class="mask" style="opacity: 0; visibility: hidden;"></div></div></div><div class="more_menu notification_menu"><div class="badge" style="display: block;">6</div><div class="wrapper"><ul><li><span><span class="message">Downloaded Movie (720p)</span><span title="Invalid Date" class="added"></span></span></li><li><span><span class="message">Downloaded Movie (720p)</span><span title="Invalid Date" class="added"></span></span></li><li><span><span class="message">Snatched "The.Book.Of.Life.2014.Repack.720p.BluRay.x264-GECKOS": Movie (2014) in 720p</span><span title="Invalid Date" class="added"></span></span></li><li><span><span class="message">Snatched "Movie.2013.720p.BluRay.x264-ROVERS": Movie (2013) in 720p</span><span title="Invalid Date" class="added"></span></span></li><li><span><span class="message">Snatched "Movie.2013.1080p.BluRay.x264-ROVERS": Movie (2013) in 1080p</span><span title="Invalid Date" class="added"></span></span></li><li><span><span class="message">Snatched "Movie.720p.BluRay.x264-CROSSBOW": Movie (2010) in 720p</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 2 releases for "Foxcatcher" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 82 releases for "The Babadook" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Downloaded Movie (BR-Rip)</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Downloaded Movie (BR-Rip)</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Snatched "Movie.2014.HDRip.XviD.AC3-EVO": Movie (2014) in brrip</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 5 releases for "Nightcrawler" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 29 releases for "Two Days, One Night" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Snatched "Movie.2014.HDRiP.XViD.AC3-H34LTH": Movie (2014) in brrip</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 16 releases for "Movie" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 23 releases for "Pride" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 21 releases for "Movie 2" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 2 releases for "Movie" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Downloaded Movie (720p)</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Snatched "Movie.2014.720p.BluRay.X264-Japhson": Movie (2014) in 720p</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Downloaded Movie? (720p)</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Snatched "Movie.2013.720p.BluRay.x264-WiKi": Movie? (2013) in 720p</span><span title="Invalid Date" class="added"></span></span></li><li><span class="read"><span class="message">Found 35 releases for "Movie" before ETA. Select and download via the dashboard.</span><span title="Invalid Date" class="added"></span></span></li></ul></div><a class="button icon2 eye-open"></a></div><div class="more_menu menu"><div class="wrapper"><ul><li><a href="http://localhost:5050/couchpotato/settings/about/">About CouchPotato</a></li><li><a>Check for Updates</a></li><li><span class="separator"></span></li><li><a href="http://localhost:5050/couchpotato/settings/general/">Settings</a></li><li><a href="http://localhost:5050/couchpotato/log/">Logs</a></li><li><span class="separator"></span></li><li><a>Restart</a></li><li><a>Shutdown</a></li></ul></div><a class="button icon2 cog"></a></div></div></div><div class="content"><div class="page home active"></div><div class="page wanted"></div><div class="page manage"></div><div class="page settings"></div><div class="page log"></div><div class="page wizard"></div><div class="page userscript"></div></div><div class="footer"></div></body></html>

**Comment by clinton-hall** [comment]
And what happens if you post
http://localhost:5050/couchpotato/api/574eb150cf9f4e1da29bcdefdcacbe3b/media.get/?id=tt2262227
into your local browser?

**Comment by slyde56** [comment]
Oooh, this looks right. Now I have what looks like a Python dictionary/tree.

**Comment by clinton-hall** [comment]
edit autoProcessMedia.cfg
```
[CouchPotato]
    [[couchpotato]]
         web_root = /couchpotato
```

See if this works... I assume you are using some for of reverse proxy?

**Comment by slyde56** [comment]
Oh my gosh, this was completely my fault. At one point I was trying to set a reverse proxy up, but I didn't get far enough and didn't change my CouchPotato Settings > General > Url Base back to blank from /couchpotato. This is what I get for mucking around with things and not changing them back...

For anyone's future reference:
```
[CouchPotato]
    [[couchpotato]]
         web_root = /couchpotato
```
must match CouchPotato Settings > General > Url Base.

Thanks so much for helping me figure this out.

**Comment by clinton-hall** [comment]
No worries. Glad it is working again.
